### ISSUE TITLE
Export of validation loss of each individual output for a multi-output model 

### ISSUE THREAD

**Issue by yfengli** [keras-team/keras#12217]
I am building a multi-output model by using keras, and like to check the loss of each individual output. 
But it seems the training returns only the mean value of losses. Is there any way that we can extract the losses of all outputs. 

Thank you!

- [ ] Check that you are up-to-date with the master branch of Keras. You can update with:
`pip install git+git://github.com/keras-team/keras.git --upgrade --no-deps`

- [ ] Check that your version of TensorFlow is up-to-date. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

**Comment by gabrieldemarmiesse** [comment]
Keras already returns the individual values in the History object. Use a jupyter notebook or Ipython to inspect it and you'll find the values you're looking for.

**Comment by yfengli** [comment]
I am new with it, is there any documentation about it so I can learn from, or how can I use callback to collect these data? Thank you!
